### ISSUE TITLE
TypeScript definitions?

### ISSUE THREAD

**Issue by danikaze** [embark-framework/neo-blessed#38]
Is there any way to use `@types/blessed` with `neo-blessed`?

**Comment by danikaze** [comment]
If using webpack, this can be done leaving
```js
import * as blessed from 'blessed';
```

and then in webpack use the resolve.alias:
```js
resolve: {
  alias: {
    blessed$: 'neo-blessed',
  },
},
```

but still... shouldn't this project have it's own definitions? since not everyone is using webpack, and if the library evolves, some new types might be needed...

**Comment by KaKi87** [comment]
Hello,

I'm using JetBrains IntelliJ to code, and I can't find any way to use `blessed`'s typings for `neo-blessed` so the autocompletion feature is down.

So, +1 for this.

Thanks

**Comment by cdaringe** [comment]
i've copied blessed's typings and swapped `declare module "blessed"` with `declare module "neo-blessed"`--works for now. would like to get them into this repo tho

**Comment by BobFrankston** [comment]
I robocopy'ed blessed to neo-blessed in @types and just updated package.json. Since this is a drop-in replacement simply copying @types/blessed to @types/neo-blessed should work but I don't have the confidence to do an official install but since it's simple, I'd encourage others to do so.

**Comment by dustinlacewell** [comment]
Hope this gets resolved.

**Comment by moonrailgun** [comment]
Is any update for this issue?

I have a local resolution for that.

- First, install `@types/blessed`
- Next, create this file in root: `types/neo-blessed/index.d.ts`
- Finally, put declare like below:
  ```
  declare module 'neo-blessed' {
    export * from 'blessed';
  }
  ```

**Comment by KaKi87** [comment]
Yes, although it won't contain any change that `neo-blessed` could have made that doesn't match `blessed`.

However, after doing a quick look on [Useful Forks](https://useful-forks.github.io/), I see that @zach-is-my-name is merging other forks, and may be trying to create a super-fork ?
